naucse_render/lesson.py

```diff
@@ -56,10 +56,15 @@ def get_lesson(lesson_slug, vars, base_path):
     lesson_vars = lesson_info.pop('vars', {})
 
     pages_info = lesson_info.pop('subpages', {})
-    pages_info.setdefault('index', {})
-
+    pages_info.setdefault('index', {'title': lesson_info['title']})
     for slug, page_info in pages_info.items():
-        info = {**lesson_info, **page_info}
+        info = {**lesson_info, 'title': None, **page_info}
+        if 'title' not in page_info:
+            try:
+                subtitle = info['subtitle']
+            except KeyError:
+                raise ValueError(f"'subtitle' is required for page {lesson_slug}/{slug}")
```

Comment:
subtitle or title?

Reply:
This is for subpages (not the index), where I'd strongly prefer `subtitle`.
Adding `title` will avoid the error. Removing the subpage will also do. But you should really only add `title` if you know what you're doing.

Reply:
ack